benchmark/pyproject.toml

```diff
@@ -9,21 +9,23 @@ packages = [{ include = "agbenchmark" }]
 
 [tool.poetry.dependencies]
 python = "^3.10"
-# helicone = "^1.0.9"  # incompatible with openai@^1.0.0
-agent-protocol-client = "^1.1.0"
+agent-protocol-client = {git = "https://github.com/Significant-Gravitas/agent-protocol.git", subdirectory = "packages/client/python"}
```

Comment:
- [ ] https://github.com/Significant-Gravitas/agent-protocol/pull/2
- [ ] Make `v1.2.0` tag after merging above PR
- [ ] Pin this dependency to `rev = "v1.2.0"` to prevent it from auto-updating to dev versions

Reply:
This *can* wait until after this PR is merged